services/queue/schemas/v1/cancel-task-group-response.yml

```diff
@@ -0,0 +1,40 @@
+$schema: "/schemas/common/metaschema.json#"
+title:              "Cancel Task-Group Response"
+description: |
+  Response from a `cancelTaskGroup` request.
+type:               object
+properties:
+  taskGroupId:
+    title:          "Task-Group Identifier"
+    description: |
+      Identifier for the task-group being listed.
+    type:           string
+    pattern:        {$const: slugid-pattern}
+  totalCount:
+    type:           integer
+    minimum:        0
+    title:          "Total number of tasks"
+    description: |
+      Total number of tasks that existed at the moment of the call.
+  cancelledCount:
+    type:           integer
+    minimum:        0
+    title:          "Total number of cancelled tasks"
+    description: |
+      Total number of tasks that were cancelled with this call.
+      It includes all non-resolved tasks with deadline in the future.
```

Comment:
Let's omit "deadline in the future" in the description and in the code. Deadline handling is the responsibility of a different component, and I think it is cleanest to uncouple it from here.